core/src/main/scala/org/broadinstitute/dsde/rawls/serviceFactory/ResourceBufferEmailManager.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.rawls.serviceFactory
+
+import org.broadinstitute.dsde.rawls.config.{RawlsConfigManager, ResourceBufferConfig}
+import org.broadinstitute.dsde.rawls.model.WorkspaceCloudPlatform.{Azure, Gcp}
+
+object ResourceBufferEmailManager {
+  def getResourceBufferEmail(appConfigManager: RawlsConfigManager): String =
+    appConfigManager.cloudProvider match {
+      case Gcp =>
+        ResourceBufferConfig(appConfigManager.conf.getConfig("resourceBuffer")).saEmail
+      case Azure =>
+        // Might need to change this to throwing exception
```

Comment:
Can you add a TODO to this comment if you plan to do it soon, or remove it if you didn't find it necessary after all?